protocol/v1/membership_update_message_test.go

```diff
@@ -223,6 +232,20 @@ func TestGroupValidateEvent(t *testing.T) {
 			Event:  NewColorChangedEvent("#7cda00", 0),
 			Result: false,
 		},
+		{
+			Name:   "image-changed allowed because from is admin",
+			From:   "0xabc",
+			Group:  createGroup([]string{"0xabc"}, nil),
+			Event:  NewImageChangedEvent([]byte{1, 2, 3}, 0),
+			Result: true,
+		},
+		{
+			Name:   "image-changed not allowed for non-admins",
```

Comment:
That's wrong. Every member should be able to change the name, image, and color of the group. This comment is just to raise awareness, no action is required. I am already resolving it in a separate issue.